hack/changelog-template.txt

```diff
@@ -9,6 +9,6 @@ HELM CHART:
 * The version of the Helm chart is now %%HELM_CHART_VERSION%%.
 
 UPGRADE:
-* For NGINX, use the %%IC_VERSION%% image from our DockerHub: `nginx/nginx-ingress:%%IC_VERSION%%` or `nginx/nginx-ingress:%%IC_VERSION%%-alpine`
+* For NGINX, use the %%IC_VERSION%% image from our DockerHub: `nginx/nginx-ingress:%%IC_VERSION%%` or `nginx/nginx-ingress:%%IC_VERSION%%-alpine` or 'nginx/nginx-ingress:%%IC_VERSION%%-ubi
```

Comment:
too many or. please see the changelog for 1.7.0 -- https://github.com/nginxinc/kubernetes-ingress/releases/tag/v1.7.0